lms/security.py

```diff
@@ -120,19 +146,25 @@ def identity(self, request):
 
         return Identity("", [])
 
-    def permits(self, request, context, permission):
-        return _permits(self, request, context, permission)
+    def permits(self, request, _context, permission):
+        return _permits(self.identity(request), permission)
 
 
-def _permits(policy, request, _context, permission):
-    identity = policy.identity(request)
+def _permits(identity, permission):
     if identity and permission in identity.permissions:
         return Allowed("allowed")
 
     return Denied("denied")
 
 
-def _get_lti_user(request) -> LTIUser:
+def _get_launches_lti_user(request) -> LTIUser:
+    schema = LTI11AuthSchema(request).lti_user
```

Comment:
Same URLs would work with both. We only try the 1.3 if the required parameter is there.